modules/network/src/test/java/org/apache/ignite/internal/network/serialization/marshal/DefaultUserObjectMarshallerWithArbitraryObjectsTest.java

```diff
@@ -553,6 +554,13 @@ void nonLatin1InStringFieldIsSupported() throws Exception {
         assertThat(unmarshalled.value, is("щ"));
     }
 
+    @Test
+    void supportsBigInteger() throws Exception {
```

Comment:
Every other test here looks like "xxxIsSupported()", maybe we should rename it

Reply:
Renamed the method